hazelcast-jet-core/src/main/java/com/hazelcast/jet/core/Edge.java

```diff
@@ -676,7 +692,13 @@ public int getClassId() {
         /**
          * This policy sends each item to all candidate processors.
          */
-        BROADCAST
+        BROADCAST,
+        /**
+         * This policy sends each item to a single processor on each of the
+         * cluster members in a round robin fashion. It is only available
+         * on a distributed edge.
+         */
```

Comment:
```suggestion
        /**
         * This policy sends an item to all members, but only to one processor on
         * each member. It's a combination of {@link #BROADCAST} and {@link
         * #UNICAST}: an item is first <em>broadcast</em> to all members, and then,
         * on each member, it is <em>unicast</em> to one processor.
         * <p>
         * If the destination local parallelism is 1, the behavior is equal to
         * {@link #BROADCAST}.
         *
         * @since 4.4
         */
```